rootfs/startup.sh

```diff
@@ -43,21 +47,33 @@ cp -r /root/{.gtkrc-2.0,.asoundrc} ${HOME}
 chown -R $USER:$USER $HOME/.[^.]*
 sed -i -e "s|%USER%|$USER|" -e "s|%HOME%|$HOME|" /etc/supervisor/conf.d/supervisord.conf
 
+# check for 'mydata' working directory
+if [ ! -d "$HOME/mydata/" ]; then
+    echo "* Not able to find $HOME/mydata/. Exiting."
+    exit 1
+fi
+
 # home folder
 if [ ! -x "$HOME/.config/pcmanfm/LXDE/" ]; then
+    echo "* creating and configuring $HOME/.config/pcmanfm/LXDE/"
     mkdir -p $HOME/.config/pcmanfm/LXDE/
     ln -sf /usr/local/share/doro-lxde-wallpapers/desktop-items-0.conf $HOME/.config/pcmanfm/LXDE/
-    chown -R $USER:$USER $HOME
+    chown -R $USER:$USER $HOME/.config/pcmanfm/LXDE
 fi
 
 # nginx workers
 sed -i 's|worker_processes .*|worker_processes 1;|' /etc/nginx/nginx.conf
 
 # nginx ssl
-if [ -n "$SSL_PORT" ] && [ -e "/etc/nginx/ssl/nginx.key" ]; then
-    echo "* enable SSL"
-	sed -i 's|#_SSL_PORT_#\(.*\)443\(.*\)|\1'$SSL_PORT'\2|' /etc/nginx/sites-enabled/default
-	sed -i 's|#_SSL_PORT_#||' /etc/nginx/sites-enabled/default
+if [ -n "$SSL_PORT" ]; then
```

Comment:
could we configure ssl in the app that uses this base? or are we going to have several base images for each ssl cert

Reply:
fixed